### PR TITLE
collapsed book and cancel menus for customer

### PR DESCRIPTION
**Pull request by John-Merritt**

### CODE REVIEW

File: HotelBooking/Views/customer_view.py [hunk 1]
```diff
@@ -25,7 +26,8 @@
 class CustomerView(View):
     user_id: str
     view_options: List[Tuple[str, View]] = [
-        ("Book/Cancel reservation", ReservedRoomsView),
+        ("Book reservation", BookReservationView),
+        ("Cancel reservation", CancelReservationView),
```
Comment (mclhtay) [inline HotelBooking/Views/customer_view.py:30]: Is admin still able to cancel reservation?

Comment (John-Merritt) [reply HotelBooking/Views/customer_view.py:30]: checking now,  but the only change I made to admin.py was removing an unused import so I think so

Comment (John-Merritt) [reply HotelBooking/Views/customer_view.py:30]: image
looks like it still works

**Review by mclhtay — APPROVED** [review]
Approve on the basis that the author did manual UI testing.

**Pull request merged**